plugin-basics/header-requirements/index.md

```diff
@@ -29,7 +29,7 @@ Available header fields:
 - **Text Domain:** The [gettext](https://www.gnu.org/software/gettext/) text domain of the plugin. More information can be found in the [Text Domain](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#text-domains) section of the [How to Internationalize your Plugin](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/) page.
 - **Domain Path:** The domain path lets WordPress know where to find the translations. More information can be found in the [Domain Path](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#domain-path) section of the [How to Internationalize your Plugin](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/) page.
 - **Network:** Whether the plugin can only be activated network-wide. Can only be set to _true_, and should be left out when not needed.
-- **Update URI:** Allows third-party plugins to avoid accidentally being overwritten with an update of a plugin of a similar name from the WordPress.org Plugin Directory. For more info read related [dev note](https://make.wordpress.org/core/2021/06/29/introducing-update-uri-plugin-header-in-wordpress-5-8/).
+- **Update URI:** **(_Important: never use for a WordPress.org plugin_)** Allows non WordPress.org plugins to avoid accidentally being overwritten with an update of a plugin of a similar name from the WordPress.org Plugin Directory. For more info read related [dev note](https://make.wordpress.org/core/2021/06/29/introducing-update-uri-plugin-header-in-wordpress-5-8/).
```

Comment:
I'd like to suggest to change a little bit the wording to match the rest of the sentence and make that clear (as WordPress.org is usually confused with WordPress itself). 
Something like:
> **(_Important: never use for a plugin hosted in the WordPress.org Plugin Directory_)** Allows plugins not hosted on WordPress.org to...